packages/reactotron-server/src/server/reactotron/commandHandler.ts

```diff
@@ -0,0 +1,11 @@
+import { commands } from "../datastore"
```

Comment:
This and `connectionHandler` job should always be simple. Take a `reactotron` event and:

- Put it in the datastore
- Publish an event if needed